.circleci/config.yml

```diff
@@ -58,136 +26,41 @@ jobs:
-      ARCH: x64
-      LIBOQS: master
-      OPENSSL: 102
-  # x86_64-xenial (Ubuntu 16.04)
-  # liboqs nist branch
-  ssl-x86_64-xenial-liboqs-nist-openssl-111:
-    <<: *openssljob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: nist
-      OPENSSL: 111
-  ssl-x86_64-xenial-liboqs-nist-openssl-102:
-    <<: *openssljob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: nist
-      OPENSSL: 102
-      
-  ################
-  # OPENSSH JOBS #
-  ################
-  # x86_64-xenial (Ubuntu 16.04)
-  # liboqs master branch, with OpenSSL enabled for OpenSSH
-  ssh-x86_64-xenial-liboqs-master-with-openssl-with-pqauth:
-    <<: *opensshjob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: master
-      WITH_OPENSSL: true
-      WITH_PQAUTH: true
-  ssh-x86_64-xenial-liboqs-master-with-openssl-no-pqauth:
-    <<: *opensshjob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: master
-      WITH_OPENSSL: true
-      WITH_PQAUTH: false
-  # x86_64-xenial (Ubuntu 16.04)
-  # liboqs master branch, without OpenSSL enabled for OpenSSH
-  ssh-x86_64-xenial-liboqs-master-no-openssl-no-pqauth:
-    <<: *opensshjob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: master
-      WITH_OPENSSL: false
-      WITH_PQAUTH: false
-  # x86_64-xenial (Ubuntu 16.04)
-  # liboqs nist branch
-  # (nist-branch doesn't support PQ auth)
-  ssh-x86_64-xenial-liboqs-nist-with-openssl:
-    <<: *opensshjob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: nist
-      WITH_OPENSSL: true
-      WITH_PQAUTH: false
-  ssh-x86_64-xenial-liboqs-nist-no-openssl:
-    <<: *opensshjob
-    environment:
-      IMAGE: dstebila/liboqs:x86_64-xenial-0.0.2
-      ARCH: x64
-      LIBOQS: nist
-      WITH_OPENSSL: false
-      WITH_PQAUTH: false
+      PREFIX: /opt/oqssa
 
 workflows:
   version: 2
   build:
     jobs:
-      - ssl-amd64-buster-liboqs-master-openssl-111
-      - ssl-amd64-buster-liboqs-master-openssl-102
-      - ssl-amd64-buster-liboqs-nist-openssl-111
-      - ssl-amd64-buster-liboqs-nist-openssl-102
-      - ssl-x86_64-xenial-liboqs-master-openssl-111
-      - ssl-x86_64-xenial-liboqs-master-openssl-102
-      - ssl-x86_64-xenial-liboqs-nist-openssl-111
-      - ssl-x86_64-xenial-liboqs-nist-openssl-102
-      - ssh-x86_64-xenial-liboqs-master-with-openssl-with-pqauth
-      - ssh-x86_64-xenial-liboqs-master-with-openssl-no-pqauth
-      - ssh-x86_64-xenial-liboqs-master-no-openssl-no-pqauth
-      - ssh-x86_64-xenial-liboqs-nist-with-openssl
-      - ssh-x86_64-xenial-liboqs-nist-no-openssl
+      - ssl-x86_64-centos8-liboqs-master-openssl-111
+      - ssl-x86_64-centos7-liboqs-master-openssl-111
+      - ssl-x86_64-alpine-liboqs-master-openssl-111
+
```

Comment:
Why do we still have these jobs in this repo? I thought we were moving all the OpenSSL integration tests out of the testing repo.

Reply:
Well, I'd consider it prudent to have some testing for the build images before they "hit the road". That of course can be done outside CI -- but I already have seen Docker images fail in a CI image that have been build (and tested) outside of it -- admittedly those were all images _containing_ libobs... So, I'm fine removing these here.